Platformer_001/Assets/MusicGenerator.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+
+public class MusicGenerator : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		DontDestroyOnLoad (transform.gameObject);
```

Comment:
Das würde ich – mit deiner Erlaubnis – gerne erstmal als Issue öffnen und mich später darum kümmern, da ich mich erstmal auf vernünftig klingende Musik konzentrieren würde, worum es mir in diesem PR auch geht…